tools/bridge/tests/test_config.py

```diff
@@ -0,0 +1,42 @@
+import bridge.config
+
+example_logging_config = """
+[logging.root]
+level = "DEBUG"
+
+[logging.loggers."bridge.main"]
+level = "DEBUG"
+
+# web3 is too verbose with level debug
+[logging.loggers.web3]
+level = "INFO"
+
+[logging.loggers.urllib3]
+level = "INFO"
+"""
+
+
+def test_load_minimal_config(write_config, minimal_config):
+    cfg = bridge.config.load_config(write_config(minimal_config))
+    print(cfg)
```

Comment:
can be removed, no?

Reply:
I actually like these, especially when a test fails. (but somehow I always have to defend these print calls :)